src/wayfinding/components/MapView.tsx

```diff
@@ -579,13 +584,24 @@ const MapView = React.forwardRef<MapViewRef, MapViewProps>(
       return true;
     };
 
+    const _effectiveProfile = () => {
```

Comment:
Non me convence _effectiveProfile. Eu tiraría máis ben por _buildProfile, _obtainProfile, _getProfile

Reply:
Nas tres PRs chameille igual. É unha forma de resolver a compatibilidade entre o deprecado e o novo atributo. Eu non creo que faga falta cambialo.